src/cvode/cvode_ls.c

```diff
@@ -1359,8 +1385,8 @@ static int cvLsLinSys(realtype t, N_Vector y, N_Vector fy, SUNMatrix A,
     retval = cvls_mem->jac(t, y, fy, A, cvls_mem->J_data,
                            vtemp1, vtemp2, vtemp3);
     if (retval < 0) {
```

Comment:
Should we SUNCheck the flag from `cvls_mem->jac`?
```suggestion
    SUNCheckCallNoRet(retval);
    if (retval < 0) {
```